pkg/machines/components/networks/createNetworkDialog.css

```diff
@@ -0,0 +1,11 @@
+.create-network-dialog-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-row-gap: 1em;
+  grid-column-gap: 1em;
```

Comment:
These can be consolidated to `grid-gap: 1em 1em;`, which can further be simplifed as `grid-gap: 1em;`